packages/rancher-prom-to-teams/generated-changes/patch/templates/configmap.yaml.patch

```diff
@@ -0,0 +1,133 @@
++  teams.j2: {{ .Files.Get "files/teams.j2" |  quote }}
++---
++apiVersion: batch/v1
++kind: Job
++metadata:
++  namespace: {{ $namespace }}
++  {{- include "prom2teams.preNameLabel" . | nindent 2 }}
++  annotations:
++    "helm.sh/hook": pre-install
++    "helm.sh/hook-delete-policy": hook-succeeded,hook-failed,before-hook-creation
++    "helm.sh/hook-weight": "5"
++spec:
++  template:
++    metadata:
++    {{- include "prom2teams.preNameLabel" . | nindent 6 }}
++    spec:
++      serviceAccountName: {{ include "prom2teams.fullname" . }}-pre-install
++      {{- if .Values.global.preInstallHook.securityContext }}
++      securityContext:
++      {{- toYaml .Values.global.preInstallHook.securityContext | nindent 8 }}
++      {{- end }}
++      containers:
++        - name: copy-pre-install-configmap
++          image: {{ include "system_default_registry" . }}{{ .Values.global.preInstallHook.image.repository }}:{{ .Values.global.preInstallHook.image.tag }}
++          imagePullPolicy: {{ .Values.global.preInstallHook.image.pullPolicy }}
++          command:
++            - /bin/sh
++            - -c
++            - >
++              if kubectl get configmap -n {{ $namespace }} {{ include "prom2teams.fullname" . }} > /dev/null 2>&1; then
++                echo "configMap already exists"
++                exit 1
++              fi;
++              kubectl patch configmap -n {{ $namespace }} --dry-run -o yaml {{ include "prom2teams.preName" . }}
++              -p '{{ printf "{\"metadata\":{\"name\": \"%s\"}}" (include "prom2teams.fullname" .) }}'
++              | kubectl apply -f -;
++              kubectl annotate configmap -n {{ $namespace }} {{ include "prom2teams.preName" . }}
++              helm.sh/hook- helm.sh/hook-delete-policy- helm.sh/hook-weight-;
++      restartPolicy: OnFailure
++      nodeSelector: {{ include "linux-node-selector" . | nindent 8 }}
++      tolerations: {{ include "linux-node-tolerations" . | nindent 8 }}
++---
++apiVersion: rbac.authorization.k8s.io/v1
++kind: ClusterRole
++metadata:
++  {{- include "prom2teams.preNameLabelAnno" . | nindent 2 }}
++rules:
++  - apiGroups:
++      - ""
++    resources:
++      - configmaps
++    verbs: ['create', 'get', 'patch']
++  - apiGroups: ['policy']
++    resources: ['podsecuritypolicies']
++    verbs:     ['use']
++    resourceNames:
++      - {{ include "prom2teams.preName" . }}
++---
++apiVersion: rbac.authorization.k8s.io/v1
++kind: ClusterRoleBinding
++metadata:
++  {{- include "prom2teams.preNameLabelAnno" . | nindent 2 }}
++roleRef:
++  apiGroup: rbac.authorization.k8s.io
++  kind: ClusterRole
++  name: {{ include "prom2teams.preName" . }}
++subjects:
++  - kind: ServiceAccount
++    name: {{ include "prom2teams.preName" . }}
++    namespace: {{ $namespace }}
++---
++apiVersion: v1
++kind: ServiceAccount
++metadata:
++  namespace: {{ $namespace }}
++  {{- include "prom2teams.preNameLabelAnno" . | nindent 2 }}
++---
++apiVersion: policy/v1beta1
++kind: PodSecurityPolicy
```

Comment:
does this pass a CIS 1.16 scan in a hardened cluster? I can spin one up for you if you want to test, but would be nice if we can confirm that since it's an easy add.